js/classes.js

```diff
@@ -180,10 +180,14 @@ class Item {
 }
 
 class Module {
+    #levelTextElement

```

Comment:
nope, bitte nicht. Das habe ich schon beim Battle auseinandersortiert. Keine HTML Element als properties in den Klassen, und keine UI manipulationen in den Klassen, so lange wir nicht klar getrennt haben, was gespeichert wird und was nicht. Rendering/UI anpassungen passieren in den Update functions in main. Die Klassen können einen Identifier zum Auffinden des UI Element haben oder meinetwegen auch ne Methode, um das Element selbst irgendwie zu finden.

Das kann man alles anders machen, aber dann sollten wir uns drüber unterhalten, wie das Rendering insgesamt funktioniert. Aktuell läufts so wie beschrieben und da andere Pattern einzuführen führt zu voll der Verwirrung.

Reply:
korrektur: modules werden noch gar nicht geladen oder gespeichert, nur current. würde das eigentlich direkt im neuen system machen wollen, da die dinger halt 4 layer an schachtelungen darstellen.

Reply:
ALso ja, ich würde gerne iwann das UI nicht mehr mit jedem Tick komplett neu befüllen. Aber wie überall auch: alles auf einmal, nicht so Insellösungen.